client/app/components/login/login.component.js

```diff
@@ -3,10 +3,9 @@ import controller from "./login.controller";
 import "./login.styl";
 
 let loginComponent = {
-  restrict: "E",
-  bindings: {},
   template,
-  controller
+  controller,
+  controllerAs: "ctrl"
```

Comment:
@nicksellen `controllerAs` in a component defaults to "ctrl", so I would suggest to leave this away (as before). I think we should accepts the implicit behaviour.
https://toddmotto.com/exploring-the-angular-1-5-component-method/